### DISCUSSION TITLE
ONNX support for tensorflow Resizing "area" method

### DISCUSSION THREAD

**Discussion by ZachZhao96** [onnx/onnx#5557] [Q&A] [answered]
Hi, 

I had an error message when I tried converting my TensorFlow.Keras model to ONNX format using tf2onnx and onnx, I realized there is a Resizing layer in my model architecture, but the resizing "area" method used is not supported by ONNX. I have tried retraining the model using other resizing methods like 'Bilinear' and "Nearest". Neither of them gave me similar validation and testing results. I wonder if there is a chance to have ONNX support the ResizeArea method for tensorflow.keras.layers.Resizing and tensorflow.image.resize in the future.

Thank you

![resizing issue](https://github.com/onnx/onnx/assets/89313814/2d25c0b7-571c-42d3-94d2-119883524691)

**Comment by justinchuby** [comment]
@fatcat-z

**Comment by fatcat-z** [comment]
@ZachZhao96 , could you please file an issue under repo https://github.com/onnx/tensorflow-onnx? I will prepare a new PR to implement this op in tf2onnx converter.

**Reply by ZachZhao96** [reply, marked as answer]
Will do, thank you!
